aws/rds/variables.tf

```diff
@@ -157,4 +157,9 @@ variable "storage_encrypted" {
 variable "performance_insights_enabled" {
   type    = bool
   default = true
-}
\ No newline at end of file
+}
+
+variable "ca_cert_identifier" {
+  type    = string
+  default = "rds-ca-rsa2048-g1"
```

Comment:
Hi @matej-topolovec will this affect existing clients of the module?

Reply:
Hi @o-leolleo 

Yes, I purposely updated the certificate by default as it will expire soon:

https://aws.amazon.com/blogs/aws/rotate-your-ssl-tls-certificates-now-amazon-rds-and-amazon-aurora-expire-in-2024/

With this cmd you can double check if this requires a downtime on your rds instances:

`aws rds describe-db-engine-versions --engine postgres --engine-version <engine_version> --query 'DBEngineVersions[*].SupportsCertificateRotationWithoutRestart'`

Reply:
nice thanks for the infos!